From 42a0d28b9edb0504c5c659bb71705acc363bef9b Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:36:03 -0400
Subject: [ethernet] i40e: spelling error

Message-id: <1409658042-22689-3-git-send-email-sassmann@redhat.com>
Patchwork-id: 90196
O-Subject: [RHEL7.1 PATCH 002/281] i40e: spelling error
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jesse Brandeburg <jesse.brandeburg@intel.com>

Fix a spelling error, s/extention/extension/.

Signed-off-by: Jesse Brandeburg <jesse.brandeburg@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit ddf1d0d7715eff6cf8f04f68ca14efdbf18bf7e2)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_txrx.c b/drivers/net/ethernet/intel/i40e/i40e_txrx.c
index d4bb482..19af4ce 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_txrx.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_txrx.c
@@ -892,7 +892,7 @@ static inline void i40e_rx_checksum(struct i40e_vsi *vsi,
        rx_status & (1 << I40E_RX_DESC_STATUS_L3L4P_SHIFT)))
   return;
 
- /* likely incorrect csum if alternate IP extention headers found */
+ /* likely incorrect csum if alternate IP extension headers found */
  if (rx_status & (1 << I40E_RX_DESC_STATUS_IPV6EXADD_SHIFT))
   return;
 
-- 
1.7.1